src/main/java/az/edu/turing/domain/dao/impl/memory/FlightDaoInmemory.java

```diff
@@ -4,31 +4,40 @@
 import az.edu.turing.domain.entity.FlightEntity;
 
 import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Optional;
 
 public class FlightDaoInmemory extends FlightDao {
+    private final Map<String, FlightEntity> flightDatabase = new HashMap<>();
+
     @Override
     public FlightEntity create(FlightEntity flightEntity) {
-        return null;
+        flightDatabase.put(flightEntity.getId(), flightEntity);
+        return flightEntity;
     }
 
     @Override
     public Collection<FlightEntity> getAll() {
-        return null;
```

Comment:
Also I think it would be better to write methods in a shorter and more functional programming style using Stream API